engine/components/sql/src/main/scala/pl/touk/nussknacker/sql/db/schema/DbMetaDataProvider.scala

```diff
@@ -23,10 +23,41 @@ class DbMetaDataProvider(getConnection: () => Connection) {
       } finally statement.close()
     } finally conn.close()
   }
+
+  def getSchemaDefinition(): SchemaDefinition = {
+    val connection = getConnection()
+    try {
+      val metaData = connection.getMetaData
+      val tables = metaData.getTables(null, getSchemaName(metaData), "%", Array("TABLE").map(_.toString))
```

Comment:
map(_.toString) is redundant?

Reply:
What about views? I think we also want them (I wonder about synonyms...)

Reply:
Added